strax/run_selection.py

```diff
@@ -283,6 +283,66 @@ def define_run(self: strax.Context,
                            " run definition")
 
 
+@strax.Context.add_method
+def available_for_run(self: strax.Context,
+                      run_id: str,
+                      include_targets: ty.Union[None, list, tuple, str] = None,
+                      exclude_targets: ty.Union[None, list, tuple, str] = None,
+                      pattern_type: str = 'fnmatch') -> pd.DataFrame:
+    """
+    For a given single run, check all the targets if they are stored.
+        Excludes the target if never stored anyway.
+    :param run_id: requested run
+    :param include_targets: targets to include e.g. raw_records,
+        raw_records* or *_nv. If multiple targets (e.g. a list) is
+        provided, the target should match any of the arguments!
+    :param exclude_targets: targets to exclude e.g. raw_records,
+        raw_records* or *_nv. If multiple targets (e.g. a list) is
+        provided, the target should match none of the arguments!
```

Comment:
Maybe add a command that exclude > include. If I understand your function correct, `includ_targets='raw_records*', exclude_tragets='*records*'` would return nothing.

Reply:
Exactly, I thought about that too, however, if someone is both excluding and including the same targets, I think they should think a bit harder. It is a bit hard to guess what a given user would have meant in this case, as such, I figured it was better to be unhelpful rather than trying to guess what a user might have meant.